.github/scripts/ep_review.py

```
@@ -0,0 +1,151 @@
+    ticket_key = f"EP-{pr_number}"
+    work_dir = Path("workdir")
+    work_dir.mkdir(parents=True, exist_ok=True)
+
+    ticket = {
+        "number": int(pr_number),
+        "title": pr.get("title", ""),
+        "body": pr.get("body", ""),
+        "author": pr.get("author", {}).get("login", "unknown"),
+        "authorAssociation": "MEMBER",
+        "headRefOid": pr.get("headRefOid", head_sha),
+        "labels": [l.get("name", "") for l in pr.get("labels", [])],
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)